var/spack/repos/builtin/packages/valgrind/package.py

```diff
@@ -44,17 +44,35 @@ class Valgrind(Package):
     version('3.11.0', '4ea62074da73ae82e0162d6550d3f129')
     version('3.10.1', '60ddae962bc79e7c95cfc4667245707f')
     version('3.10.0', '7c311a72a20388aceced1aa5573ce970')
+    version('develop', svn='svn://svn.valgrind.org/valgrind/trunk')
 
-    variant('mpi', default=True, description='Activates MPI support for valgrind')
+    variant('mpi', default=True,
+            description='Activates MPI support for valgrind')
     variant('boost', default=True,
             description='Activates boost support for valgrind')
 
     depends_on('mpi', when='+mpi')
     depends_on('boost', when='+boost')
 
+    depends_on("autoconf", type='build', when='@develop')
+    depends_on("automake", type='build', when='@develop')
+    depends_on("libtool", type='build', when='@develop')
+
     def install(self, spec, prefix):
-        options = ['--prefix=%s' % prefix,
-                   '--enable-ubsan']
+        if spec.satisfies('@develop'):
+            Executable('./autogen.sh')()
+
+        options = ['--prefix=%s' % prefix]
+        if spec.satisfies('%gcc'):
```

Comment:
Maybe this predicate should be that the compiler is not clang (or the OS is not apple?). I have no idea whether this is the case, but maybe this flag works on ICC etc.